### ISSUE TITLE
Add more weapons and custom gear with progression

### ISSUE THREAD

**Issue by Code-Sploit** [Code-Sploit/BurgerCraft#5]

**Comment by HyperSpace1749** [comment]
Probably add some kinds of armor/tools that use material such as copper, amethyst and emeralds

**Comment by Code-Sploit** [comment]
> Probably add some kinds of armor/tools that use material such as copper, amethyst and emeralds

Yes i agree but i don't want any already used types of ores/materials. There should also be some sort of meta armor for each type of weapon the player is using.
